recipes-domu/domu-image-android/files/meta-xt-images-extra/recipes-extended/android/android.bb

```diff
@@ -19,6 +19,9 @@ DEPENDS += "${JDK_VER} lz4-native"
 ANDROID_CCACHE = "prebuilts/misc/linux-x86/ccache/ccache"
 ANDROID_CCACHE_SIZE = "50G"
 
+ANDROID_JACK_ADMIN = "prebuilts/sdk/tools/jack-admin"
+ANDROID_JACK_MEM_SIZE = "4g"
```

Comment:
Why `50G` but `4g` ? And why `4G` at all? Can you calculate it based on host RAM size?